cfme/common/provider.py

```diff
@@ -437,15 +433,7 @@ def _on_detail_page(self):
 
     def load_details(self, refresh=False):
```

Comment:
Bad assumption on my part that the navigate_to() would inadvertently refresh - if am_i_here() is defined, it may or may not. Fixing